equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm5Mapper.java

```diff
@@ -148,4 +151,16 @@ public interface Crm5Mapper extends CrmMapper {
     @Mapping(target="region", source = "lsc_region")
     @Mapping(target="office", source = "lsc_accountoffice")
     CrmStandardPropertiesDTO getStandardPropertiesFromModel(Crm5DetailsModel model);
+
+    @Named("convertToTimeSpentString")
+    default String convertToTimeSpentString(String endDateTime){
+        if (StringUtils.isEmpty(endDateTime)){
+            return null;
+        }
+        if (endDateTime.length() > 8){
+            return timeDifference(DEFAULT_START_DATETIME,endDateTime,ISO_DATE_PATTERN);
+        } else {
```

Comment:
I personally prefer to call DateUtil.timeDifference(...) instead of using static function import. But that's a personal preference only